src/components/CompareResults/beta/ResultsTable.tsx

```diff
@@ -64,10 +65,23 @@ function processResults(results: CompareResultsItem[]) {
 
 function ResultsTable(props: ResultsTableProps) {
   const { themeMode } = props;
-  const compareResults: CompareResultsItem[] = useAppSelector(
-    (state: RootState) => state.compareResults.data,
-  );
-  const processedResults = processResults(compareResults);
+  const allRevisionsOption =
+    Strings.components.comparisonRevisionDropdown.allRevisions;
+
+  const processedResults = useAppSelector((state) => {
+    const { data } = state.compareResults;
+    const { activeComparison } = state.comparison;
+
+    if (activeComparison === allRevisionsOption) {
```

Comment:
In a follow-up, it would be good to change this to a key instead of using the string like I explained in another comment (that was hidden because of the code change).

Reply:
[PCF-287 Change string to key for 'All revisions' option](https://mozilla-hub.atlassian.net/browse/PCF-287)